nimble_studio_game_development_suite/nimble_studio_perforce_server/assets/setup-perforce-helix-core.sh

```diff
@@ -5,6 +5,9 @@
 # Script derived from
 # https://github.com/aws-samples/game-production-in-the-cloud-example-pipeline/blob/main/assets/setup-perforce-helix-core.sh
 
+# stop ssm agent if active to prevent interuptions from patchign 
```

Comment:
nit: Small typo with `interruptions` and `patching`

Reply:
@giordzaws fixed the typos with my last commit.